### ISSUE TITLE
Windows Installer now forces "Register with Windows"

### ISSUE THREAD

**Issue by nancym** [qutebrowser/qutebrowser#5120]
**Version info (see `:version`)**: Windows v1.8.2

**Description**: From v1.7.0 to v1.8.1 it was possible to check "Open Default Browser Settings" but uncheck "Register with Windows" but this is not possible in the v.1.8.2 Windows Installer.

Was this intentional or is it a bug? If it's intentional, I will change what I wrote on 
[qutebrowser Tips](https://www.ii.com/qutebrowser-tips/), which currently says:

> As of qutebrowser v1.7.0 the Windows installer gives you the option to set qutebrowser as your default browser and/or register it with Windows. 

Thanks for qutebrowser!

PS - I think all I will need to change on my tips page is replace "and/or" with "and" so it's an easy fix in my writing. I just want to make sure this change is here to stay.

**Comment by The-Compiler** [comment]
Hmm, that's odd. The only thing that changed in the Windows installer there was a typo fix: 7b9d257d786ae4f3caf581ba10812f76fdb020e2, so I don't think that behavior changed in any way. 
I think the behavior also depends on whether it already was registered previously - I'd guess if you run e.g. the v1.8.1 installer again, you'd see the same?

FWIW, I don't think it makes sense to allow "Open Default Browser Settings" without "Register with Windows", as the browser needs to be registered in order to set it as default.

To find the related code, search for `SectionDefaultBrowser` in [misc/nsis/install.nsh](https://github.com/qutebrowser/qutebrowser/blob/v1.8.2/misc/nsis/install.nsh) - but unfortunately, NSIS code is quite hard to follow (at least in my opinion). @bitraid could you shed some light on this?

Thanks for the nice writings! @guckes discovered them and sent me a mail just yesterday.

**Comment by bitraid** [comment]
>FWIW, I don't think it makes sense to allow "Open Default Browser Settings" without "Register with Windows", as the browser needs to be registered in order to set it as default.

This. If  "Open Default Browser Settings" is checked ("Set  as Default Browser" on Win7) the "Register with Windows" is auto-checked and locked. @nancym, i think you might remember this the other way around, and you might want to change the text to something like:
>As of qutebrowser v1.7.0 the Windows installer gives you the option to register qutebrowser with Windows and/or set it as your default browser.

**Comment by nancym** [comment]
Thanks for your thoughts @The-Compiler and @bitraid. You're right that it doesn't make sense to have it the way I think[*] it used to be and I've updated my writing.

That's cool that @guckes told you about my qutebrowser writing - I remember him from Usenet days!

[*] I actually tested this with various versions on various machines and I still think there was a change, but it's not worth any of our time to research this IMHO 😵
